src/locales/de/pokemon.json

```diff
@@ -0,0 +1,902 @@
+{
+  "alt": {
+    "mega-sep": "-"
+  },
```

Comment:
Maybe we should find a way of telling the translators what is missing? Currently, they need to cross reference the english translations with their translations, to see what they are missing to fully translate it.
I would suggest that we for example add all the 'alt'-translations to each language file, but just keep the english translation for each. Then translators from the different countries just need to open their language file, and scan it for english words.

Each time we add new translations in the future, we will then add the key-value-pair to each language-file.